api-contracts/openapi/components/schemas/v1/filter.yaml

```diff
@@ -0,0 +1,62 @@
+V1Filter:
+  properties:
+    metadata:
+      $ref: "../metadata.yaml#/APIResourceMeta"
+    tenantId:
+      type: string
+      description: The ID of the tenant associated with this filter.
+    workflowId:
+      type: string
+      format: uuid
+      minLength: 36
+      maxLength: 36
+      description: The workflow id associated with this filter.
+    resourceHint:
```

Comment:
nit on naming: I know that we're trying to keep consistency with our internal naming, but I really think should be `scope` or `partition` (have a slight preference towards the former), because I don't think `resourceHint` is very intuitive

Reply:
heh I prefer `partition` but am happy either way, agree that `resourceHint` isn't very helpful - will fix this one